.changeset/pretty-olives-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-sites': minor
```

Comment:
As this package is still on `0.1.x` this should be a `patch` to align with SemVer as it's not a breaking change.

```suggestion
'@backstage/plugin-azure-sites': patch
```